feat: Added getAssetStore for cross platform asset storage.

M3TIOR opened the pull request:
So, long story short I'm stupid. I spent a full week on this crap
trying to make an asset manager class with loading features.
It's a tiny push now, not what I was intending it to be... But
I'm glad it's done. KISS: Keep It Simple Stupid :expressionless:

M3TIOR committed:
feat: Added getAssetStore for cross platform asset storage.

So, long story short I'm stupid. I spent a full week on this crap
trying to make an asset manager class with loading features.
It's a tiny push now, not what I was intending it to be... But
I'm glad it's done. KISS: Keep It Simple Stupid :expressionless:

vyomfadia commented on Quartz/Engine/Source/Utilities/AssetManagement.cpp:
```diff
@@ -0,0 +1,115 @@
+using namespace qz::utils;
+
+// Horizontal namespace shrinking to improve code readability.
+#define FS std::filesystem
```
Maybe use ``using fs = std::filesystem`` instead.

M3TIOR committed:
refactor (asset-management): Added beeper's suggested syntax change.

I actually didn't know about the `using` keywords declarative
invocation. Thanks for pointing that out @beeperdeeper089 I'll
keep a note of that for future occurrences.

M3TIOR committed:
style (asset-management): Whoops, forgot to change the FS case...

M3TIOR commented:
Whelp... just realized that should have been a `fix` because the prior commit broke everything. Oh well, better luck next time I suppose.
